scripts/dts/extract/globals.py

```diff
@@ -331,6 +331,44 @@ def build_cell_array(prop_array):
 
     return ret_array
 
+def gpio_map_lookup(cell_parent, gpio_index):
+    index = 0
+    map_array = []
```

Comment:
These two are only used at the end of the function. Might be clearer to move them there, right before where they're used.

Then the documentation for the function ends up at the top too.

Reply:
Done